install_config/registry/extended_registry_configuration.adoc

```diff
@@ -186,20 +186,20 @@ openshift:
     secret: <secret>
 ----
 
-. Create a new secret called *registry-config* from your custom registry
-configuration file you edited locally:
+. Create a `*ConfigMap*` holding the content of each file in this directory:
 +
 ----
-$ oc secrets new registry-config config.yml=</path/to/custom/registry/config.yml>
+$ oc create configmap registry-config \
+    --from-file=</path/to/custom/registry/config.yml>/
 ----
 
-. Add the *registry-config* secret as a volume to the registry's deployment
+. Add the *game-config* ConfigMap as a volume to the registry's deployment
 configuration to mount the custom configuration file at
 *_/etc/docker/registry/_*:
 +
 ----
-$ oc volume dc/docker-registry --add --type=secret \
-    --secret-name=registry-config -m /etc/docker/registry/
+$ oc volume dc/docker-registry --add --type=configmap \
+    --secret-name=<secret-name> -m /etc/docker/registry/
```

Comment:
`s/--secret-name/--configmap-name/`. Also `<secret-name>` should be `registry-config` for a sake of consistency.